README.md

```diff
@@ -1 +1,45 @@
-# Freelancer App 
\ No newline at end of file
+# Freelancer App 
+
+
+[![CircleCI](https://circleci.com/gh/Rhuansantos/freelancer-app/tree/master.svg?style=svg)](https://circleci.com/gh/Rhuansantos/freelancer-app/tree/master)
+
+## Installation
+
+`npm install or yarn install`
+
+
+## Runing application
+
+`npm start or yarn start`
+
+## Roadmap
+
+#### Stage 1
+
+Initial proof-of-concept. [IN PROGRESS]
+
+- [x] Search
+- [ ] Implement Tests with Mocha
+- [ ] Adapt old code for the Aribnb Style Guide
+- [ ] Renderer ReactElements to simple DOM
+
+
+#### Stage 2
+
+- [ ] Back End with Elastich Search + MongoDb
+
+
+#### Stage 3
+
+- [ ] Amamzon Alexa API
+
+
+## Contributions
+
+**Before submitting a pull request,** please make sure the following is done:
```

Comment:
I would suggest adding more supportive links to help people be able to seek help if they need it.

Like:
 [how to fork](https://help.github.com/articles/fork-a-repo/)
and
[how to make pull request](https://help.github.com/articles/creating-a-pull-request/)